Assets/Scripts/Planets/PlanetSpawn.cs

```diff
@@ -1,91 +1,55 @@
     void Awake()
     {
         GameSettings.planetPrefabs = Resources.LoadAll<GameObject>("Planets");
-        
+        planetParent = GameObject.Find("Planet Parent");
         SpawningPlanet();
     }
 
     private void SpawningPlanet()
     {
-        planetParent = GameObject.Find("Planet Parent");
-        //however many x number of planets
-        for (int i = 0; i < spawnCount; i++)
+        //SpawnCount is the desired number of planets
+        for (int i = 0; i < SPAWNCOUNT; i++)
         {
-            do //one iteration of a potential planet spawn
-            {
-                //does the random spawn stuff
-                spawnX = Random.Range(-spawnRange, spawnRange);
-                spawnY = Random.Range(-YZone, YZone);
-                spawnZ = Random.Range(-spawnRange, spawnRange);
-                randScale = Random.Range(100, 300);
-                loopCounter++;
-                
-                if (planets.Count == 0) //if the list is empty
-
-                {
-                    isNotCollision = true; //set isnotcollided to true
-                }
-                else //if the list is not empty
-                {
-                    for (int j = 0; j < planets.Count; j++)//checks through the list
-                    {
-                        // This will check through the entire list of currently spawned planets
-                        // If any of them are in range then the loop will continue without setting it to true
-                        if (Mathf.Abs(spawnX - planets[j].transform.position.x) > 100
-                        && Mathf.Abs(spawnY - planets[j].transform.position.y) > 100
-                        && Mathf.Abs(spawnZ - planets[j].transform.position.z) > 100) //and compares the distance
-                        {
-                            isNotCollision = true;
-                        }
-                        else
-                        {
-                            isNotCollision = false; //ensures that even if it's set to true, it gets reset back to false when the for loop breaks
-                            j = planets.Count; //forces it to end the loop
-                        }
-                    }
-                }
-
-                //prevents infinite loop
-                if (loopCounter >= 10)
-                {
-                    break;
-                }
-
-            }while (!isNotCollision); //if its not true then it will try again
-            //TODO add loop counter to while loop
-            loopCounter = 0;
-            planetTemp = Instantiate(planetPrefab, new Vector3(spawnX, spawnY, spawnZ), Quaternion.identity);
-            planetTemp.transform.localScale = new Vector3(randScale, randScale, randScale);
-            planetTemp.transform.parent = planetParent.transform;
+            xDistance = i * ORBITAREA + BUFFER; //Set xDistance
+            scale = Random.Range(SCALEMIN, SCALEMAX); //Random scale
+            planetTemp = Instantiate(
```

Comment:
Could you either make this one line or make all the other lines (like random range on line 49) expanded like you've done here. For consistency sake :)

Reply:
This is done automatically through [CSharpier](https://csharpier.com/), the program we are using for code formatting, once a list of arguments reaches a certain length.